machine/machine.py

```diff
@@ -86,7 +86,7 @@ def version(self):
         match = self._match(cmd, regexp)
         return match.group(1)
 
-    def create(self, name, driver='virtualbox', blocking=True):
+    def create(self, name, driver='virtualbox', blocking=True, xarg=[]):
```

Comment:
Thanks for your PR!

This is not a good idea:

http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments

Reply:
yes, you are right. Done the changes as you suggested.